socorro/schemas/crash_report.json

```diff
@@ -324,6 +324,76 @@
             "type": ["integer", "null"],
             "description": "Length of time between the previous crash submission and this one. Low values indicate repeated crashes."
         },
+        "memory_measures": {
+            "type": "object",
+            "description": "Memory measurements collected or computed from the memory_report.",
+            "properties": {
+                "explicit": {
+                    "type": ["integer", "null"],
+                    "description": "The 'explicit' measurement from about:memory."
```

Comment:
Come to think of it, probably best to change "from about:memory" to "from the memory report". You could perhaps add "See about:memory for a fuller description."

Likewise for all the other fields.